### ISSUE TITLE
Clashing definitions of get_codon_alphabet

### ISSUE THREAD

**Issue by peterjc** [biopython/biopython#1162]
While looking at #1160, I noticed if the user wanted to define their own codon alphabet (e.g. for ambiguous RNA or DNA), there is a problem with ``get_codon_alphabet`` being redefined:

https://github.com/biopython/biopython/blob/biopython-169/Bio/codonalign/codonalphabet.py#L24

```python
def get_codon_alphabet(alphabet, gap="-", stop="*"):
```

vs:

https://github.com/biopython/biopython/blob/biopython-169/Bio/codonalign/codonalphabet.py#L54

``` python
def get_codon_alphabet(codon_table, gap_char="-"):
```

@zruan Can these have different names?

**Comment by zruan** [comment]
By looking at the code, I think the first `def get_codon_alphabet(alphabet, gap="-", stop="*"):` is not used anywhere else in the code. Right after the function, a `default_alphabet` is defined but not being used as well.

This might be the conflict piece of code when I do the git rebase and both copy gets retained. Simply removing the first `def get_codon_alphabet` and `get_codon_alphabet` should fix the problem.

**Comment by peterjc** [comment]
OK - can you make that change as a pull request please (you can probably do it via the GitHub web-interface). This would be easier for me to review and credits you properly. Thanks!

**Comment by zruan** [comment]
The pull request #1179 is made.

**Comment by peterjc** [comment]
Closed by #1179, thank you.
